.gitignore

```diff
@@ -39,3 +39,4 @@ setup/win32/static/postgresql*.exe
 /man/
 /share/
 /src/
+/v14env/
```

Comment:
This is unnecessary.

Reply:
But I don't want this directory ti be pushed, what else can I do?

Reply:
You could either change the config of your environnements or alternatively, have a look here to see if something helps : https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally